cdap-docs/_common/vars

```diff
@@ -0,0 +1,11 @@
+BUILD_TEMP="build-temp"
+COMMON="_common"
+COMMON_HTACCESS="$COMMON/htaccess"
+COMMON_IMAGES="$COMMON/_images"
+COMMON_SOURCE="$COMMON/_source"
+COMMON_CONF_PY="$COMMON/common_conf.py"
+COMMON_HIGHLEVEL_PY="$COMMON/highlevel_conf.py"
+COMMON_PLACEHOLDER="$COMMON/_source/placeholder_index._rst"
+HTACCESS="htaccess"
+
+MANUALS="admin-manual developers-manual examples-manual integrations reference-manual"
```

Comment:
I don't think this should be here. This is not common to the other files; only the high-level build needs to know this. If it is here, it will be hard to find, as when a new manual is added. Such as I am currently doing.